tests/test_models/test_base_model.py

```diff
@@ -2,16 +2,18 @@
 """Test BaseModel for expected behavior and documentation"""
 from datetime import datetime
 import inspect
-import models
+from models.base_model import BaseModel
 import pep8 as pycodestyle
 import time
-import unittest
+import json
+import os
+from unittest import TestCase
 from unittest import mock
-BaseModel = models.base_model.BaseModel
+import models
 module_doc = models.base_model.__doc__
```

Comment:
I believe we can merge this with line 5 to be `from models.base_model import BaseModel, __doc__ as module_doc`. If we do, we can also remove `import models` (line 12).

Reply:
Nice!  I didn't think we could (previously tried with console.__ doc__), but since models is a module, this works!